Makefile

```diff
@@ -1,3 +1,28 @@
+PKG=github.com/tonistiigi/buildx
+VERSION=$(shell git describe --match 'v[0-9]*' --dirty='.m' --always --tags)
+REVISION=$(shell git rev-parse HEAD)$(shell if ! git diff --no-ext-diff --quiet --exit-code; then echo .m; fi)
```

Comment:
Do we need an option to override these?

Reply:
They're overridable if you want:
```
make VERSION=dev REVISION=dev plugin
```

I can add support to override with an environment variable if wanted